<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AndroidDeployQtWidget</class>
 <widget class="QWidget" name="AndroidDeployQtWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="advancedActions">
     <property name="title">
      <string>Deploy options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="uninstallPreviousPackage">
        <property name="text">
         <string>Uninstall previous package</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetDefaultDevices">
        <property name="text">
         <string>Reset Default Devices</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cleanLibsPushButton">
        <property name="text">
         <string>Clean Temporary Libraries Directory on Device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="installMinistroButton">
        <property name="text">
         <string>Install Ministro from APK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
